lib/webrtc/getstats.js

```diff
@@ -329,9 +605,21 @@ function chromeOrSafariGetTrackStats(peerConnection, track, options) {
       }, null, reject);
     });
   }
-  return peerConnection.getStats(track).then(response => {
-    return standardizeChromeOrSafariStats(response, options);
-  });
```

Comment:
Do we also need to update this [code section](https://github.com/twilio/twilio-video.js/blob/master/lib/preflight/getCombinedConnectionStats.ts#L35) since it's using pc.getStats()

Reply:
@charliesantos We don't have support for custom WebRTC overrides for preflight, that is the reason why I left that section as it is. However, we could consider supporting WebRTC overrides for preflight.